CMakeLists.txt

```diff
@@ -22,11 +22,21 @@ endif()
 
 project(DALI CUDA CXX C)
 set(DALI_ROOT ${PROJECT_SOURCE_DIR})
+set(CUDA_VERSION "${CMAKE_CUDA_COMPILER_VERSION}")
 
 # Build options
 option(BUILD_TEST "Build googletest test suite" ON)
 option(BUILD_BENCHMARK "Build benchmark suite" ON)
-option(BUILD_NVTX "Build with NVTX profiling enabled" OFF)
+# if BUILD_NVTX is empty remove it and let is be default
+if ("${BUILD_NVTX}" STREQUAL "")
+  unset(BUILD_NVTX CACHE)
```

Comment:
Does it work? According to CMake docs, adding CACHE clears the cache variable _instead_ of the normal one, not in addition to.